src/wovnio/wovnphp/API.php

```diff
@@ -91,6 +93,18 @@ public static function translate($store, $headers, $original_content)
         }
     }
 
+    private static function getUriRepresentation($uri, $store, $lang)
+    {
+        $urlPatternName = $store->settings['url_pattern_name'];
+        $customDomainLangs = $store->getCustomDomainLangs();
```

Comment:
move this inside the custom domain block, even if you need to nest another if afterwards. I'm not liking how this property is being checked for null. Can we ensure that `getCustomDomainLangs` is defined if the pattern is custom_domain?

Reply:
We can enforce such a requirement but that would go against PHP's philosophy of "anything is better than nothing". Right now WOVN.php chugs along even if nothing is configured. If we choose to "halt' WOVN.php if a required setting is not fulfilled we'll have to make some big changes and make use of exceptions etc, which I don't think fits into the scope of this sprint.